data.json

```diff
@@ -55,6 +55,14 @@
     "image": "https://cdn-images-1.medium.com/max/1600/1*kQxL8yhs0c1x1WVXnjhrIw.png",
     "tags": ["clothing", "hacktoberfest", "stickers"]
   },
+  {
+    "name": "Kong",
+    "difficulty": "medium",
+    "description": "Everyone who gets 1 PR accepted to a Kong repo can get a sweet Kong Contributor T-shirt!",
+    "reference": "https://discuss.konghq.com/t/contribute-to-kong-for-hacktoberfest/2029",
+    "image": "https://user-images.githubusercontent.com/121322/45907730-f6a80b00-bdad-11e8-93ef-774392192716.png",
```

Comment:
I don't think using the hacktoberfest logo is suitable because there's an implication that's how the shirt looks.

I did some digging and it looks like they're offering this shirt:

https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt

If someone could double verify that would be awesome 👌

Reply:
Thanks for digging into this!
Kong has a separate [blog post for Hacktoberfest](https://discuss.konghq.com/t/contribute-to-kong-for-hacktoberfest/2029) mentioning "a sweet Kong Contributor T-shirt" and this one seems to be from the general contribution guidelines. Could these be different?

Open to modifying the PR or adding a separate PR depending on what the community feels.

Reply:
You can modify this PR directly :wink:

Reply:
I meant:
I can add a separate PR if the community feels that Kong's Hacktoberfest and general contribution t-shirt giveaways are independent.
Or, if the community feels that both essentially the same or related, I will modify this PR itself. 

Apologies for the ambiguity earlier.

Reply:
Modified this PR anyway.